packages/next-server/server/next-server.ts

```diff
@@ -266,8 +280,25 @@ export default class Server {
       return
     }
 
-    const resolver = interopDefault(require(resolverFunction))
-    resolver(req, res)
+    try {
+      // Parsing of cookies
+      req.cookies = parseCookies(req.headers.cookie || '')
+      // Parsing query string
+      req.query = parseQuery(req)
+      // // Parsing of body
+      req.body = await parseBody(req)
+      res.send = (statusCode, data) => sendData(res, statusCode, data)
```

Comment:
I guess we talked about this already, but this doesn't mimic the express API right 🤔 Maybe we should actually support `res.status()` though it's sort of an unfortunate api 🤔

Reply:
Maybe it makes sense to switch the arguments?
```js
res.send = (data, statusCode = 200) => ...
```
It would make `statusCode` optional in a natural way, which it probably is in most cases. Additionally, it would create some nice duality with a `redirect` method that you should probably also consider to add to these extensions 🙂 
```js
res.redirect = (url, statusCode = 301) => ...
```

Reply:
I don't want to invent a new API.

Reply:
`res.redirect` is not a new API https://expressjs.com/en/4x/api.html#res.redirect